app.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/stellar/go/support/log"
+	"github.com/stellar/starbridge/httpx"
+)
+
+type App struct {
+	httpServer *httpx.Server
+
+	prometheusRegistry *prometheus.Registry
+}
+
+type Config struct {
+	Port      uint16
+	AdminPort uint16
+}
+
+func NewApp(config Config) *App {
+	app := &App{
+		prometheusRegistry: prometheus.NewRegistry(),
+	}
+
+	app.initHTTP(config)
+	app.initLogger()
+	app.initPrometheus()
+
+	return app
+}
+
+func (a *App) initPrometheus() {
+	a.httpServer.RegisterMetrics(a.prometheusRegistry)
+}
+
+func (a *App) initLogger() {
+	log.SetLevel(log.InfoLevel)
```

Comment:
FYI you can pass the logger around as a dependency instead of using the global logger. It's something we did in the webauth and recoverysigner apps. You can create a new one with:
```go
logger := log.New()
```

Reply:
`log.New()` creates a logger with default `WARN` level. I'd need to update it to `INFO` every time. For now I'll use default logger but I think it's a good idea to update default logging level to `INFO` in `support/log`.

Reply:
Yeah let's change the default. In every app were manually changing it to info anyway. 
https://github.com/stellar/go/blob/9a2f0980ba9ed3a3d693319128c40f15cce776a7/exp/services/recoverysigner/main.go#L11-L12